milestone2.ts

```diff
@@ -164,7 +191,14 @@ let getVolunteerInfo: GetVolunteerInfo = (vol) => {
 /* these criteria, return undefined
  */
 
-let daBigTest: GetVolunteer; // Code here
+type DaBigTest = (data: Volunteer[]) => Volunteer | undefined;
+let daBigTest: GetVolunteer = (data) => {
+  let fromSlo = data.filter(x => x.city === "SLO" && x.age > 40);
+  let first = fromSlo[0];
+  let firstClone = {...first}
+  firstClone.position = "staff";
+  return firstClone;
+}; 
```

Comment:
This isn't working because you forgot to handle the case where no one could meet the conditions, in which case you would want to return undefined. you would want a check to see if you found anything with the conditions (if you use .find() it will return undefined if it doesn't find anything, as opposed to .filter() which returns an empty array)